start.html

```diff
@@ -355,7 +556,114 @@ <h3 class="accordion">
 			   can complete their installation of
 			   pbench-agent as described above. The public
 			   key should be added to the authorized-keys
```

Comment:
You might want to change this to the exact spelling used for that file, `authorized_keys`.

Reply:
This section needs  a rewrite: it is quite a mishmash.